microk8s-resources/actions/enable.storage.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "Enabling default storage class"
+sudo mkdir -p ${SNAP_COMMON}/default-storage
+cat "${SNAP}/actions/storage.yaml" | \
+sed 's@\$SNAP_COMMON@'"$SNAP_COMMON"'@g' | \
+"$SNAP/kubectl" "--kubeconfig=$SNAP/client.config" apply -f -
+echo "Storage will be available soon"
+echo "Check progress with: microk8s.kubectl get all -n kube-system"
```

Comment:
I don't think this message is necessary. We don't do it for dns and dashboard and Id rather we actually blocked the command untill it verified it was done (with a timeout) or matched what exists today